startup_scripts/075_rack_groups.py

```diff
@@ -0,0 +1,33 @@
+from dcim.models import Site,RackGroup
+from ruamel.yaml import YAML
+
+from pathlib import Path
+import sys
+
+file = Path('/opt/netbox/initializers/rack_groups.yml')
+if not file.is_file():
+  sys.exit()
+
+with file.open('r') as stream:
+  yaml=YAML(typ='safe')
+  rack_groups= yaml.load(stream)
+
+  optional_assocs = {
```

Comment:
I was not successful on creating a rack_group that did not belong to a site.
Therefore the `site` does not seem to be optional.

Hence this should be a `required_assocs`.
See https://github.com/netbox-community/netbox-docker/blob/develop/startup_scripts/080_racks.py#L17
and https://github.com/netbox-community/netbox-docker/blob/develop/startup_scripts/080_racks.py#L31-L35
for how this must be implemented.